hack/init-next-helm-chart.sh

```diff
@@ -7,7 +7,7 @@ helmChartDirName="helm-charts"
 filesToBeUpdated=("Chart.yaml" "charts/safe/Chart.yaml" "charts/sentinel/Chart.yaml" \
 "charts/spire/Chart.yaml")
 
-if [ "$1" == "-h" ]; then
+if [[ "$1" == "--h" || "$1" == "--help" ]]; then
```

Comment:
I believe we should keep using one hyphen for one letter short options and two hyphens for long options
```suggestion
if [[ "$1" == "-h" || "$1" == "--help" ]]; then
```

Reply:
Agree, addressed!!